pgmq-extension/sql/pgmq.sql

```diff
@@ -772,3 +772,70 @@ BEGIN
     PERFORM pgmq.create_non_partitioned(queue_name);
 END;
 $$ LANGUAGE plpgsql;
+
+CREATE FUNCTION pgmq.convert_archive_partitioned(table_name TEXT,
+                                                 partition_interval TEXT DEFAULT '10000',
+                                                 retention_interval TEXT DEFAULT '100000',
+                                                 leading_partition INT DEFAULT 10)
+RETURNS void AS $$
+DECLARE
+a_table_name TEXT;
+a_table_name_old TEXT;
+qualified_a_table_name TEXT;
+qualified_a_table_name_old TEXT;
+BEGIN
+  PERFORM pgmq._ensure_pg_partman_installed();
+  SELECT FORMAT('%s', 'a_'|| table_name) INTO a_table_name;
+  SELECT FORMAT('%s', 'a_'|| table_name || '_old') INTO a_table_name_old;
+  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name) INTO qualified_a_table_name;
+  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name || '_old') INTO qualified_a_table_name_old;
```

Comment:
```suggestion
  SELECT FORMAT('%s', 'a_'|| table_name) INTO a_table_name;
  SELECT FORMAT('%s', 'a_'|| table_name || '_old') INTO a_table_name_old;
  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name) INTO qualified_a_table_name;
  SELECT FORMAT('%s.%s', 'pgmq','a_'|| table_name || '_old') INTO qualified_a_table_name_old;
```

The trick is to use `%s` for strings and `%I` for identifiers. If you're just concatenating you don't need to use format. So:

```suggestion
a_table_name TEXT := 'a_' || table_name;
a_table_name_old TEXT := 'a_'|| table_name || '_old';
qualified_a_table_name TEXT := format('%I.%I', 'pgmq', table_name);
qualified_a_table_name_old TEXT := format ('%I.%I', 'pgmq', table_name || '_old');
BEGIN
```

Reply:
was there something wrong with this suggestion?

Reply:
Sorry my bad, I didn't notice this comment.